README.md

```diff
@@ -30,6 +30,8 @@ how to install Howl from source.
 based system).
 - `C compiler`: Howl has a very small C core itself, and it embeds
 dependencies written in C.
+- If you're using Windows, you *need* to build Howl inside of
```

Comment:
Perhaps a separate page on the site would be best, we could just reference to that from the README.